add IP access list of TiDB Cloud (#11950)
pingcap/docs: tidb-cloud/tidb-cloud-ip-access-list.md

---
title: Configure IP Access List
summary: Learn how to configure IP addresses that are allowed to access your Dedicated Tier cluster.
---

# Configure IP Access List

> For each Dedicated Tier cluster in TiDB Cloud, you can configure an IP access list to filter internet traffic trying to access the cluster, which works similarly to a firewall access control list. After the configuration, only the clients or applications whose IP addresses are in the IP access list can connect to your Dedicated Tier cluster.

Reviewer: ```suggestion
For each Dedicated Tier cluster in TiDB Cloud, you can configure an IP access list to filter internet traffic trying to access the cluster, which works similarly to a firewall access control list. After the configuration, only the clients and applications whose IP addresses are in the IP access list can connect to your Dedicated Tier cluster.
```

> **Note:**
>
> Configuring the IP access list is only available for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.

For a Dedicated Tier cluster, you can configure its IP access list in either of the following ways:

> - [Configure an IP access list in standard connection](#configure-ip-access-list-in-standard-connection)
> 
> - [Configure an IP access list in security settings](#configure-ip-access-list-in-security-settings)

Reviewer 2: ```suggestion
- [Configure an IP access list in standard connection](#configure-an-ip-access-list-in-standard-connection)

- [Configure an IP access list in security settings](#configure-an-ip-access-list-in-security-settings)
```

## Configure IP access list in standard connection

> To configure the IP access list for your Dedicated cluster in standard connection, take the following steps:

Reviewer 2: ```suggestion
To configure the IP access list for your Dedicated Tier cluster in standard connection, take the following steps:
```

> To configure the IP access list for your Dedicated Tier cluster in standard connection, take the following steps:

Reviewer: ```suggestion
To configure an IP access list for your Dedicated Tier cluster in standard connection, take the following steps:
```

1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
2. In the row of your Dedicated Tier cluster, click **...** and select **Connect**. A dialog is displayed.
3. In the dialog, locate **Step 1: Create traffic filter** on the **Standard Connection** tab and configure the IP access list.

    - If the IP access list of your cluster has not been set, you can click **Add My Current IP Address** to add your current IP address to the IP access list, and then click **Add Item** to add more IP addresses if necessary. Next, click **Update Filter** to save the configuration.

        > **Note:**
        >
        > For each Dedicated Tier cluster, you can add up to 7 IP addresses to the IP access list. To apply for a quota to add more IP addresses, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).

    - If the IP access list of your cluster has been set, click **Edit** to add, edit, or remove IP addresses, and then click **Update Filter** to save the configuration.

>     - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly risky.

Reviewer: ```suggestion
    - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely, which is highly risky.
```

>     - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly dangerous.

Reviewer 3: ```suggestion
    - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly risky.
```

> 

Reviewer 3: 需要把 remove 的操作加到这里吗？

Reviewer 2: remove 的操作在 Line 34: If the IP access list of your cluster has been set, click **Edit** to add, edit, or remove IP addresses, and then click **Update Filter** to save the configuration.

> ## Configure IP access list in security settings

Reviewer: ```suggestion
## Configure an IP access list in security settings
```

> ## Configure IP access list in security settings

Reviewer 3: 这个方法的操作步骤似乎更少，是不是可以和 standard connection 方法的位置对调。

Reviewer 2: standard connection 方式放到前面，是因为配置完 IP，马上就可以参考界面 connect 到集群

Author: Standard Connection 命名不准确啊，和IP Access 是一样功能，需要先配置IP Access，在配置Private link和TLS Connect，我们需要集中在这里提供IP Access 配置啊

> To configure the IP access list for your Dedicated cluster in security settings, take the following steps:

Reviewer 2: ```suggestion
To configure the IP access list for your Dedicated Tier cluster in security settings, take the following steps:
```

> To configure the IP access list for your Dedicated Tier cluster in security settings, take the following steps:

Reviewer: ```suggestion
To configure an IP access list for your Dedicated Tier cluster in security settings, take the following steps:
```

1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
2. In the row of your Dedicated Tier cluster, click **...** and select **Security Settings**. A security setting dialog is displayed.
3. In the dialog, configure the IP access list as follows:

    - To add your current IP address to the IP access list, click **Add My Current IP Address**.

    - To add an IP address to the IP access list, enter the IP address and description, and click **Add to IP List**.

        > **Note:**
        >
        > For each Dedicated Tier cluster, you can add up to 7 IP addresses to the IP access list. To apply for a quota to add more IP addresses, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).

>     - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly risky.

Reviewer: ```suggestion
    - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely, which is highly risky.
```

>     - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly dangerous.

Reviewer 3: ```suggestion
    - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely and is highly risky.
```

>     - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**, and then click **Update Filter**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely, which is highly risky.

Reviewer 2: ```suggestion
    - To allow any IP address to access your cluster (not recommended), click **Allow Access From Anywhere**. According to security best practices, it is NOT recommended that you allow any IP address to access your cluster, as this would expose your cluster to the internet completely, which is highly risky.
```

>     - To remove an IP address from the access list, click **Remove** in the line of the IP address.

Reviewer 2: ```suggestion
    - To remove an IP address from the access list, click **Remove** in the line of the IP address.

4. Click **Apply** to save the configuration.
```

---
title: Configure IP Access List

> summary: Learn how to configure the root password and allowed IP addresses to connect to your dedicated cluster.

Reviewer 4: Has the configuration for the root password been moved elsewhere?

Author: yes，Updated

---

# Configure IP Access List

IP access lists are similar to firewall access control lists, the main purpose is to filter internet traffic accessing your Dedicated tier cluster.

> 

Reviewer 2: ```suggestion

> **Note:**
>
> Configuring the IP access list is only available for Dedicated Tier clusters.
```

> In your Dedicated Tier, you have two options where you can set up IP access lists that only allow your clients or applications with specified IP addresses to connect to your Dedicated Tier cluster.

Reviewer 2: ```suggestion
For a Dedicated Tier cluster, you can configure its IP access list in either of the following ways:
```

> 1. You can click **Connect** to enter the **Standard Connection** page to configure your IP access list.
> 
> 2. You can click **...** at the top of your cluster page, select **Security Settings** to configure your IP access list.

Reviewer 2: ```suggestion
- Configure the IP access list in the **Connect** dialog

    To access the **Connect** dialog, go to the Clusters page of your project, click ... in the row of your cluster, and then select Connect.

- Configure the IP access list in security settings

    To access security settings, go to the Clusters page of your project, click ... in the row of your cluster, and then select Security Settings.
```

> > **Note:**
> >
> > Only supports configuring the IP access list of the database cluster in the Dedicated Tier, and only set IP access list for your dedicated database Cluster. 
> 

Reviewer 2: ```suggestion
```

> ## Configure IP access list in standard connection 
> 
> In the TiDB Cloud console, navigate to your Dedicated Tier [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.You can click **Connect** to enter the **Standard Connection** page to configure the IP access list.
> 
> ## Configure IP access list in security settings（Optional） 
> 
> For your Dedicated Tier clusters, you can also click the **...** in the upper-right corner of the page, select **Security Settings**, and configure IP Access List,too.
> 
> > **Tip:**
> >
> > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.

Reviewer 2: ```suggestion
```
